doc/bc/changes-8.0.md

```diff
@@ -61,6 +61,92 @@ Changes affecting version compatibility with former or future versions.
     * `eZ\Publish\Core\MVC\Symfony\Matcher\ContentMatcherFactory`
     * `eZ\Publish\Core\MVC\Symfony\Matcher\LocationMatcherFactory`
 
+* The following obsolete tables have been removed from the database schema:
+    - ezapprove_items,
+    - ezbasket,
+    - ezcollab_group,
+    - ezcollab_item,
+    - ezcollab_item_group_link,
+    - ezcollab_item_message_link,
+    - ezcollab_item_participant_link,
+    - ezcollab_item_status,
+    - ezcollab_notification_rule,
+    - ezcollab_profile,
+    - ezcollab_simple_message,
+    - ezcomment,
+    - ezcomment_notification,
+    - ezcomment_subscriber,
+    - ezcomment_subscription,
+    - ezcontentbrowserecent,
+    - ezcurrencydata,
+    - ezdiscountrule,
+    - ezdiscountsubrule,
+    - ezdiscountsubrule_value,
+    - ezenumobjectvalue,
+    - ezenumvalue,
+    - ezforgot_password,
+    - ezgeneral_digest_user_settings,
+    - ezinfocollection,
+    - ezinfocollection_attribute,
+    - ezisbn_group,
+    - ezisbn_group_range,
+    - ezisbn_registrant_range,
```

Comment:
These three belongs to ISBN FieldType _(missing use of it is most likely missing implementation around ISBN handling)_, so should probably rather be removed if that one is removed too.

Reply:
> These three belongs to ISBN FieldType _(missing use of it is most likely missing implementation around ISBN handling)_, so should probably rather be removed if that one is removed too.

AFAICS ISBN storage has been completely moved to the core table (`ezcontentobject_attribute.*`), so I'm not sure what exactly is missing. For me this FT is working fine, but I've asked QA to test this separately when testing entire PR later on, just to make sure. 

We can always safely add again any table if we implement missing feature.

_However I've realized that I didn't check `cleandata.sql` for any existing inserts, so I'll update the PR_